DEVELOPER.md

```diff
@@ -84,7 +84,7 @@ We also have a link script, such that you can test components that are in develo
 The mongodb.design website will automatically import the `*.story.tsx` file from its installed package directory to render its live example. By default, the first exported story from the `*.story.tsx` file will be rendered. To specify a different story to be rendered, define the following in the Storybook file's Meta object:
 
 ```
-import { StoryMetaType } from '@leafygreen-ui/lib';
+import { StoryMetaType } from '@lg-tools/storybook-utils';
```

Comment:
This file, and all changes in `packages/` are swapping `/lib` imports for `/storybook-utils`